examples/di/InfiAgent-DABench/run_InfiAgent-DABench.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import argparse
+import asyncio
+import json
+
+from DABench import DABench
+
+from metagpt.roles.di.data_interpreter import DataInterpreter
+
+
+def init_agent(*args, **kwargs):
+    return
+
+
+async def get_prediction(agent_class, requirement):
+    """Helper function to get prediction from a new instance of the agent"""
+    try:
+        agent = agent_class  # Instantiate the agent inside this function to avoid memory conflicts
+        result = await agent.run(requirement)
+        prediction_json = json.loads(str(result).split("Current Plan")[1].split("## Current Task")[0])
+        prediction = prediction_json[-1]["result"]
+        return prediction
+    except Exception as e:
+        print(f"Error processing requirement: {requirement}. Error: {e}")
+        return None
+
+
+async def evaluate_all(agent_class):
+    """Evaluate all tasks in DABench using the specified baseline agent"""
+    DA = DABench()
```

Comment:
Variable naming should comply with python specifications

Reply:
done